daemon/daemon_windows.go

```diff
@@ -126,8 +127,8 @@ func verifyPlatformContainerResources(resources *containertypes.Resources, isHyp
 		return warnings, fmt.Errorf("range of CPUs is from 0.01 to %d.00, as there are only %d CPUs available", sysinfo.NumCPU(), sysinfo.NumCPU())
 	}
 
-	osv := system.GetOSVersion()
-	if resources.NanoCPUs > 0 && isHyperv && osv.Build < 16175 {
```

Comment:
This version was somewhere between "RS2" and "RS3", so I picked `< RS3`

Reply:
Looking at https://en.wikipedia.org/wiki/Windows_10_version_history 16170 is "RS3" and `16170 < 16175`. However if it indeed RS3, won't it fail the check `osversion.Build() < osversion.RS3`?

Reply:
The preview builds are no longer supported afaik, so yes, just looking at GA versions.

RS3 looks to be 16299; https://github.com/microsoft/hcsshim/blob/master/osversion/windowsbuilds.go#L14 

Which wikipedia also lists;

> Version 1709 (Fall Creators Update)
The Windows 10 Fall Creators Update[465] (also known as version 1709 and codenamed "Redstone 3") is the fourth major update to Windows 10 and the third in a series of updates under the Redstone codenames. It carries the build number 10.0.16299.